### PR TITLE
Add coverage to pytwin

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**

### CODE REVIEW

**Review by lboucin — APPROVED** [review]
Change reviewed. Thanks a lot @Revathyvenugopal162 !

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -96,6 +96,16 @@ jobs:
         env:
           ANSYSLMD_LICENSE_FILE: 1055@${{ secrets.LICENSE_SERVER }}
 
+      - name: Upload coverage artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.python-version }}_${{ matrix.os }}_pytest.xml
+          path: .cov/coverage.xml
```
Comment (MaxJPRey) [inline .github/workflows/ci.yml:103]: Shouldn't it be html here?

Comment (MaxJPRey) [inline .github/workflows/ci.yml:103]:
```suggestion
          path: .cov/html 
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -96,6 +96,16 @@ jobs:
         env:
           ANSYSLMD_LICENSE_FILE: 1055@${{ secrets.LICENSE_SERVER }}
 
+      - name: Upload coverage artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.python-version }}_${{ matrix.os }}_pytest.xml
```
Comment (MaxJPRey) [inline .github/workflows/ci.yml:102]:
```suggestion
          name: ${{ matrix.python-version }}_${{ matrix.os }}_pytest.html
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Revathyvenugopal162) [reply .github/workflows/ci.yml:103]: Yes, Here the pytest is running with pyansys/actions with extra args , not from toml, adapting to html everywhere.

**Pull request merged**